keep/api/models/db/topology.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime
+from typing import List, Optional
+
+from pydantic import BaseModel
+from sqlalchemy import DateTime, ForeignKey
+from sqlmodel import JSON, Column, Field, Relationship, SQLModel, func
+
+
+class TopologyService(SQLModel, table=True):
+    id: Optional[int] = Field(primary_key=True, default=None)
+    tenant_id: str = Field(sa_column=Column(ForeignKey("tenant.id")))
+    source_provider_id: str = "unknown"
+    repository: Optional[str]
+    tags: Optional[List[str]] = Field(sa_column=Column(JSON))
+    service: str
+    environment: str = Field(default="unknown")
+    display_name: str
+    description: Optional[str]
+    team: Optional[str]
+    application: Optional[str]
+    email: Optional[str]
+    slack: Optional[str]
```

Comment:
I'm a bit worried we'll generate mess here without a strict validation. May be a channel id, or a full link...

Reply:
wdym? can you elaborate further?

Reply:
is it specifically about slack or the datamodel itself? this is what I found useful as something we can use for ui + correlation

Reply:
I can add url validation if that makes sense (although I would just go for the mess now and fix it when we get to such case), @Matvey-Kuk wdyt?